packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -190,6 +190,19 @@ async function runOnSimulator(xcodeProject, scheme, args: FlagsT) {
 }
 
 async function runOnDevice(selectedDevice, scheme, xcodeProject, args: FlagsT) {
+  const isIOSDeployInstalled = child_process.spawnSync(
+    'ios-deploy',
+    ['--version'],
+    {encoding: 'utf8'},
+  );
+
+  if (isIOSDeployInstalled.error) {
+    throw new Error(`Failed to install the app on the device because we couldn't execute the "ios-deploy" command.
+      Please install it by running "${chalk.bold(
```

Comment:
we generally don't do indentations in our errors, unless it's something very specific. So in this case it looks like you could use `CLIError` :D